src/Relationship/SubJoinWith.php

```diff
@@ -19,6 +19,8 @@ class SubJoinWith
 
     protected $select;
 
+    protected $nativeAlias;
```

Comment:
I would actually say that it needs to be marked as internal but I think all the visibilities will change with the next version anyway.

Reply:
Yea I went with `protected` since all properties are already that.